programs/server/config.yaml

```diff
@@ -975,7 +975,7 @@ settings:
     stream:
         default_shards: 1
         default_replicas: 1
-        default_sharding_expr: "rand()"
+        default_sharding_expr: ""   # Default is `rand()`, or `sip_hash64(<primary_keys...>)` if has primary keys
```

Comment:
change the comment：“Empty string means ‘rand（）’ or ’sip_hash64(<primay keys...>) if primary key set‘